web/styles/left_sidebar.css

```diff
@@ -1365,29 +1365,23 @@ li.topic-list-item {
     }
 
     #hide-more-direct-messages {
-        display: block;
+        display: grid;
+        grid-template-columns: inherit;
+        grid-template-areas: inherit;
+        grid-column: 1 / -1;
         text-decoration: none;
+        line-height: var(--line-height-sidebar-row);
         color: inherit;
         /* 12px at 14px/1em */
-        font-size: 0.8571em;
 
         & span {
```

Comment:
Can we add a class on this span? My recollection is that having a base HTML element like `span` as the last selector in a CSS rule like this results in performance problems; I started a thread in #frontend about the concern.

Reply:
Fixed in the first commit.